e2e/_suites/fleet/features/system_integration.feature

```diff
@@ -0,0 +1,224 @@
+@deploy-system_integration
+Feature: System Integration package metrics
```

Comment:
There is a preference on naming feature files with same name as the feature:
```suggestion
Feature: System Integration
```
Or you can name the file `system_integration_package_metrics.feature`

Reply:
Done